backends-velox/src/main/scala/io/glutenproject/backendsapi/velox/VeloxBackend.scala

```diff
@@ -42,6 +43,7 @@ object VeloxBackendSettings extends BackendSettings {
         // Unsupported types are prevented.
         fields.map(_.dataType).collect {
           case _: BooleanType =>
+//          case DateType =>
```

Comment:
How should we handle DateType? Looks a little confusing here.

Reply:
I forget to delete this line, deleted.